CHANGELOG.md

```diff
@@ -34,6 +35,17 @@ All notable changes to this project will be documented in this file.
     * #### Removed
 * ### NI-DCPower
     * #### Added
+        * `channel` repeated capability - See #737 for discussion
+    * #### Changed
+        * Metadata updated to NI-DCPower 17.6.1
+    * #### Removed
+* ### NI-FGEN
+    * #### Added
+        * Repeated capablilites - See #737 for discussion:
+            * `channel` repeated capability
+            * `p2p_streams` repeated capability
```

Comment:
Since #724 is not merged yet, nor is it decided upon, you should remove p2p_streams from this PR.